DQM/SiStripCommon/src/SiStripHistoId.cc

```diff
@@ -80,64 +80,49 @@ std::string SiStripHistoId::createHistoLayer(std::string description,
 
 //std::string SiStripHistoId::getSubdetid(uint32_t id, const TrackerTopology* tTopo, bool flag_ring, bool flag_thickness){
 std::string SiStripHistoId::getSubdetid(uint32_t id, const TrackerTopology *tTopo, bool flag_ring) {
-  const int buf_len = 50;
-  char temp_str[buf_len];
-
+  std::ostringstream oss;
```

Comment:
this is rather expensive.. perhaps a string and std::to_string(int) is sufficient for the use cases below?

Reply:
thanks for the feedback, addressed in the last push.